screens/Settings/SettingScreen.tsx

```diff
@@ -63,8 +65,35 @@ export const SettingScreen: React.FC<SettingProps & RootRouteProps> = (
         headerTitle={t('header')}
         headerElevation={2}
         onDismiss={controller.TOGGLE_SETTINGS}>
-        <ScrollView>
-          <Column fill backgroundColor={Theme.Colors.lightGreyBackgroundColor}>
+        <ScrollView backgroundColor={Theme.Colors.lightGreyBackgroundColor}>
+          <Text weight="semibold" margin="10" color={Theme.Colors.aboutVersion}>
+            {t('injiAsVerifierApp')}
+          </Text>
+          <Row
+            align="space-evenly"
+            backgroundColor={Theme.Colors.whiteBackgroundColor}>
+            <Pressable onPress={controller.RECEIVE_CARD}>
+              <Column style={Theme.Styles.receiveCardsContainer}>
+                <Image
+                  source={Theme.ReceiveCardIcon}
+                  style={{ alignSelf: 'center' }}
+                />
+                <Text margin="6" weight="semibold">
+                  {t('receiveCard')}
+                </Text>
+              </Column>
+            </Pressable>
+            <ReceivedCards
+              isVisible={false}
+              service={undefined}
+              vcItemActor={undefined}
```

Comment:
If these properties aren't defined, will this not be `undefined` by default? In another way, is there any good-practice/reason to define a property as `undefined` explicitly?

NOTE: I'm speaking about this by strictly thinking from Javascript perspective & not JSX/React perspective, so I might be wrong here.

Reply:
Here in ReceivedCards.tsx I'm passing/using "HomeScreenProps" in the controller which is already created for HomeScreenTab in the previous UI of the app.
As of then, we had "Received cards in Home Screen".
If removed or changed the value of props it my affect the home screen, VC's, and Expansion of VC... etc